### PR TITLE
feat: move form stepper to island-ui

### PR DESCRIPTION
**Pull request by jeremybarbet**
# feat: move form stepper to island-ui

— 

## What

Move form stepper to island-ui

## Why

This component is part of the design system so we want it to be available in the island-ui and storybook

## Screenshots / Gifs

Screenshot 2020-10-02 at 11 20 32

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Comment by oddsson** [comment]
According to the design the blue progress bar at the top should not be there.

Screenshot 2020-10-02 at 11 24 31

**Review by oddsson — COMMENTED** [review]
According to the design the blue progress bar should not be there 

Screenshot 2020-10-02 at 11 24 31

**Comment by jeremybarbet** [comment]
The blue bar is part of the storybook layout. Sorry, I should have make it more obvious 
Screenshot 2020-10-02 at 11 35 35

File: libs/application/ui-fields/src/lib/FormStepper.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    children: section.children.map((child) => formattedChildren(child)),
+  }))
+
+  const ProgressTag: FC = () => {
+    switch (mode) {
+      case FormModes.REVIEW:
+      case FormModes.PENDING:
+        return (
+          <Tag variant="darkerBlue" label bordered>
+            Status: In Review
```
Comment (madebynoam) [inline libs/application/ui-fields/src/lib/FormStepper.tsx:61]: In a future PR we should localise these strings inside the tag.

**Review by madebynoam — DISMISSED** [review]
Lgtm!

**Review by arnifreyrs — APPROVED** [review]
looks great

**Comment by arnifreyrs** [comment]
only thinking if the title should be part of the form stepper (the form name), but that could change later.

File: libs/application/ui-shell/src/components/fields/index.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export { default as TextFormField } from './TextFormField'
 export { default as FileUploadFormField } from './FileUploadFormField'
 export { default as ReviewFormField } from './ReviewFormField'
 export { default as DividerFormField } from './DividerFormField'
+export { default as FormStepper } from './FormStepper'
```
[the remark this answers is not in the document]
Comment (jeremybarbet) [reply libs/application/ui-shell/src/components/fields/index.ts]: I created my branch on an old version of master, that's why..

**Comment by jeremybarbet** [comment]
It looks to be part of the design system so it's probably best it is here

**Review by arnifreyrs — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -34,7 +34,6 @@
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "storybook:island-ui": "nx run island-ui-storybook:storybook",
```
Comment (oddsson) [inline package.json:37]: Why is this being removed?

Comment (jeremybarbet) [reply package.json:37]: I renamed the script inside the workspace directly to bind with the "serve" script, so we can do `yarn start island-ui-storybook`

**Review by arnifreyrs — DISMISSED** [review]

**Review by arnifreyrs — DISMISSED** [review]

**Review by DavidBachmann — APPROVED** [review]

**Pull request merged**